Backend/starter_project_dotnetforce7/BlogApp2/BlogApp.Application/Responses/Result.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BlogApp.Application.Responses
+{
+    public class Result<T>
+    {
+        public bool Success { get; set; } = true;
+        public string Message { get; set; }
+        public T? Value { get; set; }
+        public List<string> Errors { get; set; }
+    }
+}
```

Comment:
This looks good. For the seek of consistency make sure all other team members are utilizing this response type.